### ISSUE TITLE
Timepoint slider bug?

### ISSUE THREAD

**Issue by tischi** [BIOP/bigdataviewer-biop-tools#23]
@NicoKiaru Camilla (see E-mail) reported:

> When showing sources in a new window we did not have the option to chose more timepoints, and there is no slider on the bottom of the window to slide through the timepoints.

Do you know what this could be about?

**Comment by NicoKiaru** [comment]
> Didn't we have some code that tries to figure out how many timepoints an SAC has? In my use case it is an SAC that is derived from an ImagePlus.

Yes, the code I linked tries to figure it out in a general manner
